tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -656,10 +660,32 @@ public function userRemovesLinkFromSpaceUsingRootEndpointOfGraphAPI(
 		string $space
 	):void {
 		$this->featureContext->setResponse(
-			$this->removeAccessToSpace($sharer, 'link', $space)
+			$this->removeAccessToSpace($sharer, 'link', $space, null)
```

Comment:
`null` can be default
```suggestion
			$this->removeAccessToSpace($sharer, 'link', $space)
```